src/Compilers/VisualBasic/Test/Symbol/BasicCompilerSymbolTest.vbproj

```diff
@@ -39,6 +39,10 @@
       <Project>{7FE6B002-89D8-4298-9B1B-0B5C247DD1FD}</Project>
       <Name>CompilerTestResources</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\..\Test\Utilities\CSharp\CSharpCompilerTestUtilities.csproj">
+      <Project>{4371944a-d3ba-4b5b-8285-82e5ffc6d1f9}</Project>
+      <Name>CSharpCompilerTestUtilities</Name>
+    </ProjectReference>
```

Comment:
Is this reference necessary? There is a `CommonTestBase.CreateCSharpCompilation` method that should be available already.